This patch causes the RTEMS build process to not only
install 'librtemscpu.a/librtemsbsp.a' but also the
individual libraries they are composed of.

NOTE: you need a recent version of automake to
      regenerate 'Makefile.in'. If you configure RTEMS
      with --enable-maintainer-mode, the necessary
      steps are performed automatically.

THIS IS A PATCH AGAINST rtems-4.6.0pre2
      
Till Straumann, 2003/4/22

Index: cpukit/wrapup/Makefile.am
===================================================================
RCS file: /afs/slac/g/spear/cvsrep/rtems/src-20030128/cpukit/wrapup/Makefile.am,v
retrieving revision 1.1.1.4
diff -c -r1.1.1.4 Makefile.am
*** cpukit/wrapup/Makefile.am	21 Apr 2003 20:20:11 -0000	1.1.1.4
--- cpukit/wrapup/Makefile.am	22 Apr 2003 05:01:48 -0000
***************
*** 76,85 ****
  
  TMPINSTALL_FILES += $(project_libdir)/librtemscpu$(LIB_VARIANT).a
  
! all-am: $(ARCH) $(LIB) $(TMPINSTALL_FILES)
  
  install-data-hook: $(LIB)
  	@$(mkinstalldirs) $(DESTDIR)$(libdir)
! 	$(INSTALL_DATA) $(LIB) $(DESTDIR)$(libdir)
  
  include $(top_srcdir)/automake/local.am
--- 76,86 ----
  
  TMPINSTALL_FILES += $(project_libdir)/librtemscpu$(LIB_VARIANT).a
  
! all-am: $(ARCH) $(LIB) $(LIBS) $(TMPINSTALL_FILES)
  
+ ## TillS: install individual libs as well
  install-data-hook: $(LIB)
  	@$(mkinstalldirs) $(DESTDIR)$(libdir)
! 	$(INSTALL_DATA) $(LIB) $(LIBS) $(DESTDIR)$(libdir)
  
  include $(top_srcdir)/automake/local.am
Index: c/src/wrapup/Makefile.am
===================================================================
RCS file: /afs/slac/g/spear/cvsrep/rtems/src-20030128/c/src/wrapup/Makefile.am,v
retrieving revision 1.1.1.4
diff -c -r1.1.1.4 Makefile.am
*** c/src/wrapup/Makefile.am	21 Apr 2003 20:19:06 -0000	1.1.1.4
--- c/src/wrapup/Makefile.am	23 Apr 2003 00:03:34 -0000
***************
*** 15,22 ****
  SRCS += ../lib/libbsp/@RTEMS_CPU@/@RTEMS_BSP_FAMILY@/wrapup/$(ARCH)/libbsp.a
  endif
  
  if HAS_NETWORKING
! SRCS += ../libnetworking/wrapup/$(ARCH)/libnetapps.a
  endif
  
  if HAS_RDBG
--- 15,23 ----
  SRCS += ../lib/libbsp/@RTEMS_CPU@/@RTEMS_BSP_FAMILY@/wrapup/$(ARCH)/libbsp.a
  endif
  
+ ## TillS: omit netapps from generic lib
  if HAS_NETWORKING
! EXCLUDED = ../libnetworking/wrapup/$(ARCH)/libnetapps.a
  endif
  
  if HAS_RDBG
***************
*** 68,75 ****
  	rm -f $(ARCH)/*
  	$(RANLIB) $@
  
  install-data-local:
  	@$(mkinstalldirs) $(DESTDIR)$(bsplibdir)
! 	$(INSTALL_DATA) $(LIB) $(DESTDIR)$(bsplibdir)
  
  include $(top_srcdir)/../../automake/local.am
--- 69,78 ----
  	rm -f $(ARCH)/*
  	$(RANLIB) $@
  
+ ### TillS: install the individual libs as well
  install-data-local:
  	@$(mkinstalldirs) $(DESTDIR)$(bsplibdir)
! 	$(INSTALL_DATA) $(LIB) $(EXCLUDED) $(DESTDIR)/$(libdir) done
! 	$(INSTALL_DATA) $(SRCS) $(DESTDIR)/$(libdir) done
  
  include $(top_srcdir)/../../automake/local.am

